src/mon/MonClient.cc

```diff
@@ -625,17 +625,21 @@ MonConnection& MonClient::_add_conn(unsigned rank)
 
 void MonClient::_add_conns()
 {
-  unsigned n = cct->_conf->mon_client_hunt_parallel;
-  if (n == 0 || n > monmap.size()) {
-    n = monmap.size();
-  }
-  vector<unsigned> ranks(n);
-  for (unsigned i = 0; i < n; i++) {
+  unsigned m = monmap.size();
```

Comment:
nit, could have a better name than `m`, like `num_mons`, also could make this `const`.